lexus_rx_450h_2019/docker-compose.yml

```diff
@@ -45,6 +45,19 @@ services:
       - /opt/carma/routes:/opt/carma/routes
     command: wait-for-it.sh localhost:11311 -- roslaunch /opt/carma/vehicle/config/carma_docker.launch
 
+  lightbar-driver:
+      image: usdotfhwastol/carma-lightbar-driver:latest
```

Comment:
We will wait on merging this until the light bar component release so we can use the actual version number here